src/ontology/mondo-edit.obo

```diff
@@ -557231,6 +557238,7 @@ is_a: MONDO:0020573 {source="OMIM:300909"} ! inherited disease susceptibility
 intersection_of: MONDO:0020573 ! inherited disease susceptibility
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12823 ! XPNPEP2
 intersection_of: predisposes_towards MONDO:0015057 ! renin-angiotensin-aldosterone system-blocker-induced angioedema
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12823 ! XPNPEP2
```

Comment:
mondo.owl v2024-12-03 does not have any source provenance for this gene association